monorepo-folders/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "private": true,
+  "name": "monorepo-folders",
+  "workspaces": [
+    "packages/*"
+  ],
+  "scripts": {
+    "frontend": "yarn workspace frontend-ui start",
+    "backend": "yarn workspace backend-apis start",
+    "worker": "yarn workspace temporal-worker start",
+    "build.workflows": "yarn workspace temporal-workflows compile",
+    "start": "yarn build.workflows && concurrently --kill-others-on-fail \"yarn frontend\"  \"yarn backend\" \"yarn worker\""
```

Comment:
```
    "start": "yarn build.workflows && concurrently --kill-others-on-fail -n 'frontend,api-server,worker' \"yarn frontend\"  \"yarn backend\" \"yarn worker\""
```

Can give names to clarify logging:

![image](https://user-images.githubusercontent.com/251288/140595507-c40ca2f7-a7a9-48da-941d-29b547e6012e.png)

Reply:
wait how does that naming work?

Reply:
ok got it. good idea